runelite-client/src/main/java/net/runelite/client/plugins/chatfilter/ChatFilterPlugin.java

```diff
@@ -192,7 +192,7 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 			: COLLAPSIBLE_MESSAGETYPES.contains(chatMessageType) && config.collapseGameChat();
 		if (!blockMessage && shouldCollapse)
 		{
-			Duplicate duplicateCacheEntry = duplicateChatCache.get(name + ":" + message);
+			Duplicate duplicateCacheEntry = duplicateChatCache.get(name + ":" + messageNode.getValue());
```

Comment:
Are you sure this is correct? I think `message` always has to be `messageNode.getValue()`?

Reply:
Good callout, they're the same, so I changed back to using just `message`.